### ISSUE TITLE
Would be nice to create generic secrets dynamically

### ISSUE THREAD

**Issue by lewismarshall** [UKHomeOffice/vault-sidekick#8]
Without this feature it is hard to have self service without some bash curling or another container in pipelines to create secrets.

In the absence of a custom resource or other workflow, this feature will enable a server service to be fully self service.

The example below shows how this could be implemented as a simple additional option for at least the generic secret backend. Maybe the option should also be specific to the generic:
```
vault-sidekick -cn=secret:/namespace/secrets/db-root-password:create:true
```

@gambol99 @jon-shanks @vaijab @timgent

**Comment by vaijab** [comment]
Sounds sensible.

**Comment by allandegnan** [comment]
Happy with the idea, the UI looks a bit weird but I guess it's consistent with the vault-sidekick (also weird) so :+1:

**Comment by lewismarshall** [comment]
Fixed by #9
